creator-node/src/services/ContentNodeInfoManager.ts

```diff
@@ -170,43 +171,42 @@ export async function getReplicaSetSpIdsByUserId({
   if (blockNumber) {
     // In total, will try for 200 seconds
     const MAX_RETRIES = 201
```

Comment:
need to update comment on line 172 + significantly reduce this `MAX_RETRIES` value haha, currently this will retry for ~200 * 10min lmao, 10 is prob good